integration/droplet_snapshots_test.go

```diff
@@ -80,15 +80,17 @@ ID      Name           Type        Distribution    Slug      Public    Min Disk
     "distribution": "Fedora",
     "type": "snapshot",
     "slug": "slimey",
-    "min_disk_size": 25
+    "min_disk_size": 25,
+	"created_at": "2018-09-20T19:28:00Z"
```

Comment:
Everything looks great. 
Let's quickly replace tabs and it should be good to go :dolphin: 
Excellent job :clap: 
```suggestion
    "created_at": "2018-09-20T19:28:00Z"
```

Reply:
fair enough, for my defense I just changed my laptop and didn't set my IDE correctly :)